src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/Helpers.cs

```diff
@@ -51,36 +49,26 @@ internal static RoslynTrigger GetRoslynTrigger(EditorAsyncCompletionData.Complet
             }
         }
 
-        internal static CompletionTriggerKind GetRoslynTriggerKind(EditorAsyncCompletionData.CompletionTrigger trigger)
+        internal static CompletionTriggerKind GetRoslynTriggerKind(EditorAsyncCompletionData.CompletionTriggerReason triggerReason)
         {
-            switch (trigger.Reason)
+            return triggerReason switch
             {
-                case EditorAsyncCompletionData.CompletionTriggerReason.InvokeAndCommitIfUnique:
-                    return CompletionTriggerKind.InvokeAndCommitIfUnique;
-                case EditorAsyncCompletionData.CompletionTriggerReason.Insertion:
-                    return CompletionTriggerKind.Insertion;
-                case EditorAsyncCompletionData.CompletionTriggerReason.Deletion:
-                case EditorAsyncCompletionData.CompletionTriggerReason.Backspace:
-                    return CompletionTriggerKind.Deletion;
-                case EditorAsyncCompletionData.CompletionTriggerReason.SnippetsMode:
-                    return CompletionTriggerKind.Snippets;
-                default:
-                    return CompletionTriggerKind.Invoke;
-            }
+                EditorAsyncCompletionData.CompletionTriggerReason.InvokeAndCommitIfUnique => CompletionTriggerKind.InvokeAndCommitIfUnique,
+                EditorAsyncCompletionData.CompletionTriggerReason.Insertion => CompletionTriggerKind.Insertion,
+                EditorAsyncCompletionData.CompletionTriggerReason.Deletion or EditorAsyncCompletionData.CompletionTriggerReason.Backspace => CompletionTriggerKind.Deletion,
```

Comment:
so this seems to unify Delete/Backspace to just Delete (which is fine).  but later on i see roslyn code checkking for backspace as well... which then seems weird to me.